cli/src/install.sh

```diff
@@ -95,6 +95,17 @@ source ${completions_dir}/phylum.bash" \
     success "Completions are enabled for bash."
 }
 
+check_glibc() {
+    # On systems with musl libc, running ldd on phylum will exit with an error.
+    # If that happens, report an explanation and exit.
+    ldd phylum >/dev/null 2>&1
+    if [ $? != 0 ]; then
```

Comment:
It's much more natural to put the call directly in the if

```suggestion
    if ldd phylum >/dev/null 2>&1; then
```